engine/baml-runtime/src/internal/llm_client/primitive/aws/aws_client.rs

```diff
@@ -909,23 +1094,10 @@ impl AwsClient {
                         )
                     }
                     BamlMediaContent::Base64(b64_media) => {
-                        // AWS Bedrock supports video for Nova models with specific format
-                        let mime_type = media.mime_type_as_ok()?;
-                        let format = match mime_type.as_str() {
-                            "video/mp4" => bedrock::types::VideoFormat::Mp4,
-                            "video/mpeg" => bedrock::types::VideoFormat::Mpeg,
-                            "video/mov" => bedrock::types::VideoFormat::Mov,
-                            // "video/avi" => bedrock::types::VideoFormat::Avi,
-                            "video/x-flv" => bedrock::types::VideoFormat::Flv,
-                            "video/mkv" => bedrock::types::VideoFormat::Mkv,
-                            "video/webm" => bedrock::types::VideoFormat::Webm,
-                            _ => {
-                                anyhow::bail!(
-                                    "AWS Bedrock video format not supported: {}. Supported formats: mp4, mpeg, mov, flv, mkv, webm",
-                                    mime_type
-                                );
-                            }
-                        };
+                        // Always strip MIME prefix and let AWS validate/err.
+                        let format = bedrock::types::VideoFormat::from(strip_mime_prefix(
+                            &media.mime_type_as_ok()?,
+                        ));
```

Comment:
aaggghhh. i was wrong, we should put back the manual mime type matching - this will not delegate to the bedrock api server but to the bedrock sdk and will actually result in worse error messages.